### DISCUSSION TITLE
How to diaplay the agent name and last seen

### DISCUSSION THREAD

**Discussion by VinothGox** [chaskiq/chaskiq#634] [Q&A]
Hi Michelson,

After I set up chaskiq app on our website chat app working is fine but does not display the agent name. please help me.

I've attached a screenshot.
![Screenshot from 2022-02-09 12-36-46](https://user-images.githubusercontent.com/76249948/153140035-34f24bbf-8258-4ca7-828d-137c50940b24.png)

Regards,
Vinoth

**Comment by michelson** [comment]
Hi, did you edit the agent name and avatar? If you enter to settings -> team what do you see?

**Reply by VinothGox** [reply]
yeah, I updated the agent name and profile image. But the top of the view (i mean nav bar) does not display the agent profile image only shows the backspace icon.
![imageedit_2_9615454810](https://user-images.githubusercontent.com/76249948/153226885-c3328cb3-4b5d-4537-94b4-233cbe1eb707.jpg)

**Comment by michelson** [comment]
maybe there is no assignee, go to the conversation and assign someone

**Reply by VinothGox** [reply]
Got it, Thank you.

**Comment by VinothGox** [comment]
Get notified by email is still loading, not responded. one time I get a server error after I put the email id not displayed an error message or success message.

I've attached a screenshot.
![Screenshot from 2022-02-09 23-26-48](https://user-images.githubusercontent.com/76249948/153262000-ff78ad83-6ae5-4b30-9a3b-b9a9dec589f9.png)

**Comment by michelson** [comment]
do you have the server error trace? that looks like an error on the
backend. (the workflow was interrupted somehow)

**Comment by VinothGox** [comment]


**Reply by VinothGox** [reply]
Do you know? in the case after putting the email only loading. not displayed the "email:xxxx@gmail.com" in this model.
